src/js/plugins/plugin.search.js

```diff
@@ -330,37 +239,16 @@ BookReader.prototype.BRSearchCallbackErrorMobile = function(results) {
  * @param {jQuery} $el
  * @param {boolean} fade
  */
-BookReader.prototype._BRSearchCallbackError = function(results, $el, fade) {
-  this.$('.BRnavpos .search').remove();
-  this.$('.BRmobileSearchResultWrapper').empty(); // Empty mobile results
-
+BookReader.prototype._BRSearchCallbackError = function(results) {
   this.searchResults = results;
-  let timeout = 2000;
   if (results.error) {
-    if (/debug/.test(window.location.href)) {
```

Comment:
ahh the things we squirrel away...

Reply:
Yeah, with a sourcemap and breakpoints, we really don't need inline debugging conditionals.